ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/timestamp/LoggingTimestampExtractor.java

```diff
@@ -71,11 +70,15 @@ TimestampExtractor getDelegate() {
     return delegate;
   }
 
-  private long handleFailure(final Object value, final RuntimeException e) {
+  private long handleFailure(final Object key, final Object value, final RuntimeException e) {
+    final Object safeKey = ObjectUtils.defaultIfNull(key, "null");
     final Object safeValue = ObjectUtils.defaultIfNull(value, "null");
```

Comment:
nit: I don't think we need these `safeX` do we? string concatenation should handle it